explore-analyze/alerts-cases/alerts/alerting-getting-started.md

```diff
@@ -3,11 +3,14 @@ applies_to:
   stack: ga
   serverless: ga
 navigation_title: Getting started with alerts
+products:
+  - id: kibana
+description: Step-by-step tutorial for creating and configuring your first alerting rule.
 ---
 
 # Getting started with alerting [alerting-getting-started]
 
-Alerting enables you to define *rules*, which detect complex conditions within different {{kib}} apps and trigger actions when those conditions are met. Alerting is integrated with [**{{observability}}**](../../../solutions/observability/incident-management/alerting.md), [**Security**](detection-rules://index.md), [**Maps**](../../../explore-analyze/alerts-cases/alerts/geo-alerting.md) and [**{{ml-app}}**](../../../explore-analyze/machine-learning/anomaly-detection/ml-configuring-alerts.md). It can be centrally managed from **{{stack-manage-app}}** and provides a set of built-in [connectors](../../../deploy-manage/manage-connectors.md) and [rules](../../../explore-analyze/alerts-cases/alerts/rule-types.md#stack-rules) for you to use.
+Alerting enables you to define rules that detect complex conditions within different {{kib}} apps and trigger actions when those conditions are met. Alerting is integrated with {{observability}}, **Security**, **Maps**, and the **{{ml-app}}**, can be centrally managed from **{{stack-manage-app}}**, and provides a set of built-in connectors and rules for you to use.
```

Comment:
Couple of issues with the second sentence:
- AFAIK, the narrative that we've promoted for alerting is that alerting features are available for Security, Observability, and Kibana (Stack Management). I'm not sure why maps and ML are highlighted as well, 
- It's pretty generous to state that Security, Observability, and Kibana rules can be managed from Stack Management. Observability and Kibana rules can be fully-managed, but Security rules can't. In order to edit Security rules and access all of the functionality that's offered, you need to go to the Security app. 
- Security is the only solution that offers "built-in rules". Also, those rules (which we call "prebuilt rules" in the Security docs) cannot be managed from Stack Management.